pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -20,16 +20,19 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/go-openapi/spec"
```

Comment:
Another alternative is to have a method that takes an auth interface and returns the OpenAPISpec documentation for that. Our apiserver will use that method and hence depend on open api spec. Anyone else who just wants to use the authorizer can still do that without depending on open api spec.